pkg/kubelet/images/image_gc_manager.go

```diff
@@ -190,16 +186,9 @@ func NewImageGCManager(runtime container.Runtime, statsProvider StatsProvider, r
 func (im *realImageGCManager) Start() {
 	ctx := context.Background()
 	go wait.Until(func() {
-		// Initial detection make detected time "unknown" in the past.
-		var ts time.Time
-		if im.initialized {
-			ts = time.Now()
-		}
-		_, err := im.detectImages(ctx, ts)
+		_, err := im.detectImages(ctx, time.Now())
```

Comment:
thinking about it, previously a kubelet restart triggered a GC (which is exactly what is being fixed here) and I wonder if there are ppl out there depending on this to free disk space on nodes with high disk pressure.
However now we need to wait a bit more (basically the actual min age to expire after a restart) so it should not be a big deal even for these folks.